xblockmufi/templates/view.html

```diff
@@ -0,0 +1,61 @@
+            <li class="specialButton">
+                <a href="#" title="Toggle Keyboard for Special Characters"><i class="fa fa-keyboard-o fa-lg"></i><span class="sr">Toggle Keyboard for Special Characters<span class="toggle-open">, open</span></a>
+            </li>
+        </ul>
+        <div class="specialCharTable">
+            <div class="charZoom"><i class="fa fa-book fa-lg hint intro-zoom-icon"></i></div>
+            <div class="charZoomTitle"><span class="hint">Select a character to insert...</span></div>
+            <div class="charWrapper">
+                <div class="char" title="Lowercase 'yogh'">&#x021D;</div>
+                <div class="char" title="Capital 'yogh'">&#x021C;</div>
+                <div class="char" title="Lowercase 'wynn'">&#x01BF;</div>
+                <div class="char" title="Capital 'wynn'">&#x01F7;</div>
+                <div class="char" title="Lowercase 'eth'">&#x00F0;</div>
+                <div class="char" title="Capital 'eth'">&#x00D0;</div>
+                <div class="char" title="Lowercase 'thorn'">&#x00FE;</div>
+                <div class="char" title="Capital 'thorn'">&#x00DE;</div>
+                <div class="char" title="Lowercase 'e caudata'">&#x0119;</div>
+                <div class="char" title="Capital 'E caudata'">&#x0118;</div>
+                <div class="char" title="Lowercase '&#x00E6;sh', ligature of 'ae'">&#x00E6;</div>
+                <div class="char" title="Capital '&#x00E6;sh', ligature of 'AE'">&#x00C6;</div>
+
+                <div class="char" title="'punctus interrogativus'" style="clear: both;">&#xF160;</div>
+                <div class="char" title="'punctus flexus'">&#xF1F5;</div>
+                <div class="char" title="'punctus versus'">&#xF1EA;</div>
+                <div class="char" title="'punctus elevatus', diagonal stroke">&#xF1F0;</div>
+                <div class="char" title="'punctus elevatus'">&#xF161;</div>
+                <div class="char" title="Medieval Comma">&#xF1E0;</div>
+                <div class="char" title="'paragraphus'">&#xF1E1;</div>
+                <div class="char" title="'signe de renvoi'">&#xF1EC;</div>
+                <div class="char" title="Tironian Nota, 'et'">&#x204A;</div>
+                <div class="char" title="Sharp s, ligature of 'fz' or 'fs'">&#x00DF;</div>
+                <div class="char" title="Letter 'o' with long stroke overlay, 'obiit'">&#xA74A;</div>
+                <div class="char" title="Palm Branch, 'ramus'">&#xF1DB;</div>
+            </div>
+        </div>
+        <div class="contents" contenteditable="true">{{ student_answer }}</div>
+    </div>
+    <div class='expert_answer' style='display:none'>
+        <div class='our_answer'>{{ our_answer_label }}</div>
+        <div class='xblockmufi_block_textarea explanation'>{{ answer_string }}</div>
+    </div>
+    {% if not is_past_due %}
+        <div class='button_holder {{ submit_class }}'>
```

Comment:
let's rename the class variable to something like: `button_display_class`.

Reply:
Should I still keep the `{{ submit class }}`?

Reply:
yes, that's the variable you're renaming.

Reply:
So you want it to look like `<div class='button_holder {{ button_display_class }}'>`?